src/input-format.md

```diff
@@ -1,6 +1,19 @@
 r[input]
 # Input format
 
+r[input.syntax]
+```grammar,lexer
+@root CHAR -> <a Unicode scalar value>
+
+NUL -> U+0000
+
+TAB -> U+0009
+
+LF -> U+000A
+
+CR -> U+000D
+```
```

Comment:
This feels a bit odd to reference this as the input syntax (and put it here.  I would have expected `input.syntax` to be something like
```grammar
@root INPUT -> SHEBANG? (BLANK_LINES FRONTMATTER)? TOKEN*
```
`BLANK_LINES` might refer to part of this but not all of it.